searx/settings.yml

```diff
@@ -77,6 +77,7 @@ search:
   # formats: [html, csv, json, rss]
   formats:
     - html
+    # - json
```

Comment:
> Users can now know how to enable json without needing to search 'how-to' online.

Pew .. json, csv and rss are already listed two lines above .. in line 77 .. and why are searching for a "how-to"? .. I will never understand why people ask for outdated and faulty HowTo on the internet when there is up-to-date documentation -> https://docs.searxng.org/admin/settings/settings_search.html ..